pkgs/development/python-modules/django-ratelimit/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, django
+, django-redis
+, fetchFromGitHub
+, pymemcache
+, pythonOlder
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "django-ratelimit";
+  version = "4.1.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "jsocol";
+    repo = "django-ratelimit";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-ZMtZSKOIIRSqH6eyC7bBeua7YLKyWW6NOXN/MDv9fy4=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+  ];
+
+  propagatedBuildInputs = [
+    django
+    django-redis
+    pymemcache
+  ];
+
```

Comment:
Can we easily execute https://github.com/jsocol/django-ratelimit/blob/main/django_ratelimit/tests.py ?